backend/main/server/controllers.go

```diff
@@ -512,16 +432,52 @@ func (a *App) updateProposal(w http.ResponseWriter, r *http.Request) {
 		}
 	}
 
-	p.Status = &payload.Status
-	p.Cid, err = helpers.pinJSONToIpfs(p)
+	if *payload.Proposal.Status == "draft" {
```

Comment:
should check the actual status of the proposal in the DB, not the status submitted in the payload since it could be anything